Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -166,6 +166,25 @@ class OWTestLearners(widget.OWWidget):
     TARGET_AVERAGE = "(Average over classes)"
     class_selection = settings.ContextSetting(TARGET_AVERAGE)
 
+    class Error(OWWidget.Error):
+        class_required = Msg("Train data input requires a class variable")
+        class_required_test = Msg("Test data input requires a class variable")
+        too_many_folds = Msg("Number of folds exceeds the data size")
+        class_inconsistent = Msg("Test and train data sets "
+                                 "have different classes")
+
+    class Warning(OWWidget.Warning):
+        missing_data = Msg()
```

Comment:
I cannot decide whether I like this short syntax or not. It is shorter to write than explicitly setting "{}" as a message, but I am not sure it is obvious that the message will be supplied later.